proselint/checks/misc/but.py

```diff
@@ -14,13 +14,31 @@
 
 """
 
-from proselint.registry.checks import Check, types
+import re
+from collections.abc import Iterator
+
+from proselint.registry.checks import Check, CheckResult
+
+
+def _check_but_paragraphs(text: str, check: Check) -> Iterator[CheckResult]:
+    pattern = r"(^|^\n|\n\n)But\b"
```

Comment:
```suggestion
    pattern = r"(^\n|\n\n)But\b"
```
The content will never start without a linebreak, due to the content wrapping in `LintFile`.

Also, consider moving this outside of the function into a "constant" wrapped with `re.compile`.

Reply:
It will, in tests, I was considering whether to modify the tests to just "mock" a lint file by prepending a \n or not, but I'll let you decide